### ISSUE TITLE
WebP images shown as GIF

### ISSUE THREAD

**Issue by gsouquet** [vector-im/element-web#24015]
### Steps to reproduce

1. Send a webp image. See the GIF 

#### What did you expect?

Not to see the "GIF" toast on the top left

#### What happened instead?

<img width="355" alt="Screenshot 2022-12-15 at 09 37 27" src="https://user-images.githubusercontent.com/769871/207812048-d17955a7-cd6b-482b-8a12-a6637f4e00cc.png">

### Will you send logs?

No

**Comment by t3chguy** [comment]
Yeah IIRC some other platform does the same thing so I copied it - thought "WEBP" would just look confusing and some platforms even send "GIF"s as MP4s nowadays so the word seems to have lost all meaning.

**Comment by gsouquet** [comment]
I don't really understand why an overlay is needed here. We do not show anything for JPG or PNG. Why treat webp differently?

I can understand the argument for animated webp. But the images above are still.

Not quite sure why you added the `X-Needs-Design` label for? This sounds more like a defect to me where we should not show the `GIF` overlay at all.

**Comment by t3chguy** [comment]
It should be only shown on animated WEBP images. The overlay is there as a hint that hovering plays the animation. `blobIsAnimated` actually checks for the animated flag in the blob.

**Comment by HarHarLinks** [comment]
> maybe your image is actually a 1-frame animated WEBP image

could Element be made to check for that? I have had the same issue before. There are always going to be "improper" webps like this in the wild as this isn't usually obvious to users.

**Comment by t3chguy** [comment]
Yes it could, but writing a full WEBP parser would add a lot more complexity and edge cases to test. I couldn't find a minimal one to just import at the time of writing it initially. Contributions welcome.
